Source/Diagnostics/ComputeDiagFunctors/RhoFunctor.cpp

```diff
@@ -20,11 +20,13 @@
 
 RhoFunctor::RhoFunctor (const int lev,
                         const amrex::IntVect crse_ratio,
+                        bool apply_filter_rho,
```

Comment:
Could we rename this variable to e.g. `apply_rz_psatd_filter`, to convey more precisely what this variable does?

Reply:
I agree, fixed in 044d20727910.